src/main/java/com/google/devtools/build/lib/remote/README.md

```diff
@@ -1,170 +1,3 @@
-
-Once installed, edit nginx.conf with a section for uploading and serving cache objects.
-
-```
-location /cache/ {
-    root /some/document/root;
-    dav_methods PUT;
-    autoindex on;
-    allow all;
-    client_max_body_size 256M;
-}
-```
-
-You will need to change `/some/document/root` to a valid directory where NGINX can write to and
-read from. You may need to change `client_max_body_size` option to a larger value in case the cache
-object is too large.
-
-### Apache HTTP Server with WebDAV module
-
-Assuming Apache HTTP Server is installed with DAV modules installed. You need to edit `httpd.conf` to enable the following modules:
-
-```
-LoadModule dav_module libexec/apache2/mod_dav.so
-LoadModule dav_fs_module libexec/apache2/mod_dav_fs.so
-```
-
-Edit `httpd.conf` to use a directory for uploading and serving cache objects. You may want to edit
-this directory to include security control.
-
-```
-<Directory "/some/directory/for/cache">
-    AllowOverride None
-    Require all granted
-    Options +Indexes
-
-    Dav on
-    <Limit HEAD OPTIONS GET POST PUT DELETE>
-        Order Allow,Deny
-        Allow from all
-    </Limit>
-    <LimitExcept HEAD OPTIONS GET POST PUT DELETE>
-        Order Deny,Allow
-        Deny from all
-    </LimitExcept>
-</Directory>
-```
-
-## Remote caching using the gRPC protocol
-
-We're working on a [gRPC protocol](https://github.com/googleapis/googleapis/blob/master/google/devtools/remoteexecution/v1test/remote_execution.proto)
-that supports both remote caching and remote execution. Bazel ships with a server-side implementation that's useful for testing and not intended for production use. [Buildfarm](https://github.com/bazelbuild/bazel-buildfarm) is an open source project that aims to provide a distributed remote execution platform.
-
-### Bazel Setup
-
-In order to enable remote caching in Bazel you'll need to specify some flags. We recommend adding them to your `~/.bazelrc` file for ease of use.
-
-```
-build --spawn_strategy=remote --genrule_strategy=remote --strategy=Javac=remote --strategy=Closure=remote
-build --remote_cache=replace-with-your.host:port
-```
-
-The above will enable remote caching but with sandboxing disabled. The support for sandboxing with remote caching is currently (as of 0.9.0) experimental (but works well in our experience).
-
-```
-build --experimental_remote_spawn_cache
-build --remote_cache=replace-with-your.host:port
-```
-
-Remote execution can be enabled by specifying the `--remote_executor=replace-with-your.host:port` flag.
-
-### Running the Remote Worker
-
-Bazel currently provides a sample gRPC caching backend.
-
-```
-$ git clone https://github.com/bazelbuild/bazel.git
-$ cd bazel
-$ bazel build //src/tools/remote:worker
-$ bazel-bin/src/tools/remote/worker --listen_port=8080
-```
-
+The documentation has been moved to https://docs.bazel.build/versions/master/build-event-protocol.html.
```

Comment:
should this link to https://docs.bazel.build/versions/master/remote-caching.html instead? This is what I was looking for after this change when I went to reference the old README

Reply:
Very good point. Thanks! Fixing ...